File: ProjetASI2/Personal-Finance-Management-Spring-boot/src/main/java/com/PersonalFinanceManagementApplication/Personal/Finance/Management/Application/dataTransferObject/CategoryPersonDto.java
package com.PersonalFinanceManagementApplication.Personal.Finance.Management.Application.dataTransferObject;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CategoryPersonDto {
    BigDecimal monthlyLimit;
    Long id_category;
    Long id_person;
}

File: ProjetASI2/PersonalFinanceManagementAppFrontEnd/src/app/Model/CategoryPersonDto.ts
export class CategoryPersonDto{
    monthlyLimit:Number;
    id_category:Number;
    id_person:Number;

    constructor(monthlyLimit:Number, id_category:Number, id_person:Number){
        this.monthlyLimit = monthlyLimit;
        this.id_category = id_category;
        this.id_person = id_person;
    }
}